lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -47,7 +47,8 @@ A ``data_source`` tool contains a few more relevant attributes.
         <xs:element name="macros" type="Macros" minOccurs="0"/>
         <xs:element name="edam_topics" type="EdamTopics" minOccurs="0"/>
         <xs:element name="edam_operations" type="EdamOperations" minOccurs="0"/>
-        <xs:element name="requirements" type="Requirements" minOccurs="0"/>
+        <xs:element name="xrefs" type="xrefs" minOccurs="0" />
+	<xs:element name="requirements" type="Requirements" minOccurs="0"/>
```

Comment:
I think the change of indentation here was by mistake.

Reply:
You're right  : corrected in commit 17807f4